src/org/usfirst/frc/team694/robot/subsystems/Acquirer.java

```diff
@@ -43,10 +48,19 @@ public void deacquire() {
     }
 
     public void flipUp() {
-        acquirerSolenoid.set(true);
+        acquirerFlipSolenoid.set(true);
     }
 
     public void flipDown() {
-        acquirerSolenoid.set(false);
+        acquirerFlipSolenoid.set(false);
+        Timer.delay(0.5);
```

Comment:
I'd be careful using this. If this is the same as `Thread.sleep` then your bot will stop moving for 0.5 seconds.

I'd say don't do any timing in the subsystems. That's what command groups are for.

Reply:
Oh my, I think that is the delay thing which stops the whole robot.